js/main.js

```diff
@@ -0,0 +1,28 @@
+var $placeholder = document.querySelector('.placeholder-image');
+var $urlInput = document.querySelector('.photo');
+
+$urlInput.addEventListener('input', function () {
+  $placeholder.setAttribute('src', $urlInput.value);
```

Comment:
This works, but I would recommend using `event.target.value` to guarantee you're using the correct value